src/main/java/uk/gov/hmcts/ccd/ApplicationParams.java

```diff
@@ -152,6 +152,9 @@
     @Value("${ccd.new-access-control-enabled}")
     private boolean ccdNewAccessControlEnabled;
 
+    @Value("${enable-pseudo-role-assignments-generation}")
+    private boolean enablePseudoRoleAssignmentsGeneration;
```

Comment:
Where is the property defined **enable-pseudo-role-assignments-generation**

Reply:
it is in the application.properties